.travis.yml

```diff
@@ -0,0 +1,13 @@
+services:
+- docker
+script:
+- echo "Build the TODO list"
+- cp .env.template .env
```

Comment:
You have this step but never set any relevant values in the file - that's a good sign that building your docker images with a detached `docker-compose up` command is a little odd!
If we were to do it this way we should at least use `docker-compose build` rather than creating the containers and detaching from them, but it would be much clearer here to just build the containers we care about explicitly, e.g. `docker build --target testing-offline --tag <tag>` and then just run that container directly - that would  be clearer to fresh eyes, and allow us to remove this copy step

Reply:
Thanks Jack - corrected